data/data.json

```diff
@@ -8612,10 +8612,10 @@
   },
   "v1.20.14-rancher2-1": {
    "etcd": "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
-   "alpine": "rancher/rke-tools:v0.1.79",
```

Comment:
Note to reviewers: will add new versions to update rke-tools and nginx in a new PR after this and https://github.com/rancher/kontainer-driver-metadata/pull/809 is merged.